data/dbus/com.redhat.yggdrasil.service.in

```diff
@@ -1,3 +1,4 @@
 [D-BUS Service]
 Name=com.redhat.yggdrasil1
 Exec=@SBINDIR@/yggd
+User=root
```

Comment:
I actually want to review this system service activation concept. I think this isn't configured correctly; I think the dbus service needs to match the system service name. That should enable service activation, eliminating the need to start the systemd service explicitly.